### PR TITLE
Add new converters

### PR DESCRIPTION
**Pull request by DistortedPumpkin**
Add new command argument converters: `MemberConverter`, `MessageConverter`, `RoleConverter`, `InviteConverter`

### CODE REVIEW

**Comment by DistortedPumpkin** [comment]
Going to fix and re-create

**Comment by Cryptex-github** [comment]
No that now how u do the return type I did it wrong before !

**Comment by Cryptex-github** [comment]
You do it like Converter[Type]
class SomeConverter(Converter[SomeType])

**Comment by DistortedPumpkin** [comment]
> You do it like Converter[Type] class SomeConverter(Converter[SomeType])

Oh alright, ill edit mine and the previously existing ones now

File: ferris/plugins/commands/converters.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+from typing import TYPE_CHECKING
 
-class ChannelConverter(Converter):
-    async def convert(self, ctx: Context, argument: str) -> ConverterOutputT:
+if TYPE_CHECKING:
+    from ... import Channel, Guild, User, Member, Message, Role, Invite
+    from .parser import Converter
+    from .models import Context
+
+import re
```
Comment (Cryptex-github) [inline ferris/plugins/commands/converters.py:11]: Sort imports please

**Review by Cryptex-github — CHANGES_REQUESTED** [review]

File: ferris/plugins/commands/converters.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
-from ferris.errors import NotFound
```
Comment (Cryptex-github) [inline ferris/plugins/commands/converters.py:1]: And you need `from __future__ import `annotation` in order to use if `TYPE_CHECKING` guard

**Review by Cryptex-github — CHANGES_REQUESTED** [review]

**Review by Cryptex-github — APPROVED** [review]

**Pull request merged**